test/v1/routing/VersionRoutingRequestHandlerSpec.scala

```diff
@@ -17,6 +17,7 @@
 package v1.routing
 
 import akka.actor.ActorSystem
+import api.models.errors.{InvalidAcceptHeaderError, UnsupportedVersionError}
```

Comment:
can we move this file to match where it is in the app directory? so we'd move this to `test/routing/`